apps/ejabberd/src/ejabberd_ctl.erl

```diff
@@ -62,6 +62,8 @@
 -type format_type() :: binary() | string() | char().
 -type cmd() :: {CallString :: string(), Args :: [string()], Desc :: string()}.
 
+-define(ASCII_SPACE_CHARACTER, $\s).
```

Comment:
Addressed in 6868962, but made the change for all SPACE literals, just for consistency.